docs/serverless/06-02-gitrepository-cr.md

```diff
@@ -0,0 +1,51 @@
+---
+title: GitRepository
+type: Custom Resource
+---
+
+The `gitrepositories.serverless.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to define and manage Git repositories that store the Function's source code and dependencies. To get the up-to-date CRD and show the output in the YAML format, run this command:
+
+```bash
+kubectl get crd gitrepositories.serverless.kyma-project.io -o yaml
+```
+
+## Sample custom resource
+
+This is a sample custom resource that creates a GitRepository object pointing to a Git repository with the Function's source code and dependencies. This resource specifies that the repository requires an SSH key to authenticate to it and points to the Secret that stores these credentials.
+
+```yaml
+apiVersion: serverless.kyma-project.io/v1alpha1
+kind: GitRepository
+metadata:
+  name: sample-with-auth
+spec:
+  url: "git@github.com:sample-organization/sample-repo.git"
+  auth:
+    type: key
+    secretName: kyma-git-creds
+```
+
+## Custom resource properties
```

Comment:
```suggestion
## Custom resource parameters
```

Reply:
See the [template](https://github.com/kyma-project/community/blob/master/guidelines/templates/resources/custom-resource.md#custom-resource-parameters) for reference.